DeusEx/Classes/DXRMenuSetupRando.uc

```diff
@@ -125,32 +125,32 @@ function CreateControls()
 
     i=0;
     btnGameMode.values[i++] = "Original Story";
-    btnGameMode.values[i++] = "Original Story, Rearranged Levels (Beta)";
     btnGameMode.values[i++] = "Horde Mode (Beta)";
-    btnGameMode.values[i++] = "Kill Bob Page (Beta)";
+    btnGameMode.values[i++] = "Original Story, Rearranged Levels (Alpha)";
+    //btnGameMode.values[i++] = "Kill Bob Page (Alpha)";
     /*btnGameMode.values[i++] = "How About Some Soy Food?";
     btnGameMode.values[i++] = "Horde Mode";//defending the paris cathedral could be really cool
     btnGameMode.values[i++] = "Stick With the Prod";*/
     GameMode = CreateEnum(row++, "", "Choose a game mode!", btnGameMode);
 
     i=0;
-    btnRandoDoors.values[i++] = "Undefeatable Doors Destructible & Pickable";
+    btnRandoDoors.values[i++] = "Some Doors Destructible or Pickable";

```

Comment:
is "Some Doors Destructible or Pickable" the best default? or maybe it could be too difficult and "Some Doors Destructible & Pickable" is a better default? or "Key-Only Doors Destructible or Pickable"